Apps/WebClient/src/ClientApp/public/index.html

```diff
@@ -18,49 +18,57 @@
             name="google-site-verification"
             content="doAsS2f0kVQcQMjZcJuo2Pdj60GkhXI2iKbwfaUBImk"
         />
-
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/modernizr/2.8.3/modernizr.min.js" integrity="sha512-3n19xznO0ubPpSwYCRRBgHh63DrV+bdZfHK52b1esvId4GsfwStQNPJFjeQos2h3JwCmZl0/LgLxSKMAI55hgw==" crossorigin="anonymous">
+        </script>
         <!-- Snowplow starts plowing - Standalone vE.2.14.0 -->
         <script type="text/javascript">
-            (function (p, l, o, w, i, n, g) {
```

Comment:
good idea, I will do this one then

Reply:
@sslaws per your comment on the adding a cdn to our index, I don't have any concerns with it. @FuriousLlama may have some though. Btw, the custom JS that I generated is way smaller than its cdn and we have option to choose only the ES6 Number feature of the modernizr.js and not the whole package.

Reply:
> Since we're still working on this, an optional request would be to take the Snowplow code into its own js file and embed a reference as we're doing with the modernizr

wow, now I can make the ContentSecurityPolicy > script-src is super light, it can contain only "script-src": "https://www2.gov.bc.ca/StaticWebResources/static/sp/sp-2-14-0.js" without complaining violation due to inline execution.

Reply:
If you move it to its own file make sure that the snowplow script actually adds the scripts it wants in the position it needs

Reply:
also, there might be a difference for snowplow between runing on the header vs running from the body

Reply:
Even it seems to work find to place them in the body, I have moved scripts back to the header to avoid any side effects for snowplow

Reply:
@FuriousLlama a part of logging at the starting and ending of this blog, is there any ways that we can confirm if snowplow is working correctly?